test/csv_manipulation/test_merge_results.py

```diff
@@ -41,6 +41,52 @@ def test_merge_all_results():
     assert set(df.columns) == set(["classification", "score"] + list(weights.keys()))
 
 
+def test_merge_result_append_existing_file():
+    input_c1 = Path("./data/csv/c1/c1_result.csv")
+    input_c2 = Path("./data/csv/c2/c2_result.csv")
+    weights = {"mpap0": {"0": 1, "2": 2, "3,4": 3}, "mpap0_test": {"0": 1, "2": 2, "3,4": 3}}
```

Comment:
nitpick : ça pourrait être intéressant d'avoir le même changement que ce que j'ai fait ailleurs pour vérifier le fonctionnement quand on a des classes différentes par métrique
```suggestion
    weights = {"mpap0": {"0": 1, "2": 2, "3,4": 3}, "mpap0_test": {"0": 1, "2": 2, "5": 3}}
```